assets/vue/components/import-modal.vue

```diff
@@ -146,20 +154,32 @@
 				this.$store.commit( 'showImportModal', false )
 			},
 			startImport: function () {
+				if( this.siteData.template_name ){
+					this.runMigration();
```

Comment:
You can return false after executing the `runMigration()` method so the else is not necessary.